src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -15066,16 +14910,37 @@ void VRFCondenserEquipment::VRFHR_OU_HR_Mode(EnergyPlusData &state,
 
         // perform iterations to calculate Te at the given compressor speed and operational conditions
         {
-            Par(1) =
-                state.dataHVACVarRefFlow->VRFTU(state.dataHVACVarRefFlow->TerminalUnitList(this->ZoneTUListPtr).ZoneTUPtr(1)).VRFSysNum; // VRFCond;
-            Par(2) = CompSpdActual;
-            Par(3) = Tdischarge;
-            Par(4) = h_IU_evap_in;
-            Par(5) = h_comp_in;
-            Par(6) = Q_c_TU_PL;
-            Par(7) = m_air_evap_rated;
-
-            General::SolveRoot(state, ErrorTol, MaxIte, SolFla, Tsuction_new, VRFOUTeResidual_FluidTCtrl, Tsuction_LB, Tsuction_HB, Par);
+
+            auto f = [&state, this, CompSpdActual, Tdischarge, h_IU_evap_in, h_comp_in, Q_c_TU_PL, m_air_evap_rated](Real64 const Te) {
+                int VRFCond = state.dataHVACVarRefFlow->VRFTU(state.dataHVACVarRefFlow->TerminalUnitList(this->ZoneTUListPtr).ZoneTUPtr(1))
+                                  .VRFSysNum; // VRFCond;
+
+                Real64 Ncomp_temp;   // compressor power [W]
+                Real64 Q_c_tot_temp; // total evaporator load, including piping loss [W]
+                Real64 Q_c_OU_temp;  // OU evaporator load, including piping loss [W]
+                Real64 Te_new;       // newly calculated OU evaporating temperature
+                Real64 Tfs;          // OU evaporator coil surface temperature [C]
+
+                state.dataHVACVarRefFlow->VRF(VRFCond).VRFOU_CompCap(
```

Comment:
Definite candidate here for returning a struct rather than a bunch of reference parameters.  That isn't needed here, but should be done soon.